css-view-transitions-1/Overview.bs

```diff
@@ -833,28 +830,36 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		The [=view-transition layer=] is a sibling of this stacking context.
 
 	- The [=view-transition layer=] paints after the stacking context for the [=document element=] and [/=document=]'s [=Document/top layer=].
+		This includes the filters and effects that are applied to the [=document element=].
 
 	Note: The intent of the feature is to be able to capture the contents of the page, which includes the top layer elements.
 	In order to accomplish that, the [=view-transition layer=] cannot be a part of the captured top layer context,
 	since that results in a circular dependency.
 	Instead, this stacking context is a sibling of other page contents.
 
-	Issue: Do we need to clarify that the stacking context for the root and top layer elements has filters and effects coming from the [=document element=]'s style?
-
 ## [=Captured elements=] ## {#captured-elements}
 
 	A <dfn>captured element</dfn> is a [=struct=] with the following:
 
 	<dl dfn-for="captured element">
 		: <dfn>old image</dfn>
-		:: an image or null. Initially null.
+		:: an {{ImageData}} or null. Initially null.
+
+		: <dfn>old width</dfn>
+		: <dfn>old height</dfn>
+		:: an {{unrestricted double}}, initially zero.
 
-			Issue: The type of "image" needs to be linked or defined.
+		: <dfn>old transform</dfn>
+		:: a <a data-xref-type="css-type">&lt;transform-function&gt;</a>, initially the [=identity transform function=].
 
-		: <dfn>old styles</dfn>
-		:: a set of styles or null. Initially null.
+		: <dfn>old object-view-box</dfn>
```

Comment:
Actually, we should remove this. Based on the resolution at https://github.com/w3c/csswg-drafts/issues/8597, UA doesn't add a `object-view-box` style anymore. It's supposed to be done transparently by the implementation.

Reply:
Right. removed all viewbox stuff.